### PR TITLE
Pauli et al. 2017 atlas

### PR DESCRIPTION
**Pull request by Gilles86**
Hey guys,

I made a commit that includes a possibility for downloading the subcortical atlas of Pauli et al., 2017. Appreciate feedback.

Cheers,
Gilles

### CODE REVIEW

File: nilearn/datasets/atlas.py [hunk 1]
```diff
@@ -1116,3 +1116,73 @@ def fetch_atlas_talairach(level_name, data_dir=None, verbose=1):
...
+
+    files = [(filename,
+              url_maps,
+              {'move':filename}),
+             ('labels.txt',
+              url_labels,
+              {'move':'labels.txt'})]
+    atlas_file, labels = _fetch_files(data_dir, files)
+
+    atlas_img = check_niimg(atlas_file)
```
Comment (GaelVaroquaux) [inline nilearn/datasets/atlas.py]: This loads the atlas, doesn't it? We tend to prefer to return the filename.

**Comment by GaelVaroquaux** [comment]
Overall looks good. Thank you.

I made a small comment.

Also, can you add the atlas to one of the examples, for instance to http://nilearn.github.io/auto_examples/01_plotting/plot_prob_atlas.html

**Comment by Gilles86** [comment]
Cool. I revised code to your suggestion. Also included the example.

**Comment by bthirion** [comment]
Beautiful, thx !

**Comment by GaelVaroquaux** [comment]
I forgot: can you add an entry to docs/whats_new.rst and to docs/modules/reference.rst

File: nilearn/datasets/atlas.py [hunk 2]
```diff
@@ -1116,3 +1116,71 @@ def fetch_atlas_talairach(level_name, data_dir=None, verbose=1):
     description = _get_dataset_descr(
         'talairach_atlas').decode('utf-8').format(level_name)
     return Bunch(maps=atlas_img, labels=labels, description=description)
+
```
Comment (GaelVaroquaux) [inline nilearn/datasets/atlas.py]: Pep8 tells us that we need two empty lines here.

**Pull request merged**